core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"go.uber.org/multierr"
 
+	clienttypes "github.com/smartcontractkit/chainlink/common/client"
 	txmgrtypes "github.com/smartcontractkit/chainlink/common/txmgr/types"
```

Comment:
I didn't realize that we have been building under `common`. Did we consider `core/chains`? That seems more fitting to me.